README.md

```diff
@@ -81,8 +140,10 @@ To create a new frontend container, use this command. Replace ```<MyContainer>``
 ```bash
 npx elsie generate container --pathname <MyContainer>
 ```
+> **Location:**  
+> Generated files will be placed in `src/components/`, `src/containers/`, and `src/api/` respectively
```

Comment:
Is this added in by accident due to a merge conflict, as this also exists on line 156? If this is intensional, that's fine too.